### DISCUSSION TITLE
How use Remix when Building also mobile App

### DISCUSSION THREAD

**Discussion by pullmann4rent** [remix-run/remix#6733] [Q&A]
Hello,

I have read about BFF and in Every article They speak about microservice and how to do with Web App ,iOS and android so every Need this own service so how can I Make it with Remix ? When I get much request from mobile users and Web Users how can I say both have Their own Server ?

**Comment by sergiodxa** [comment]
You can use Remix to expose an API to your mobile app, or if your Remix app already consumes an API you can use that and skip Remix.
